### PR TITLE
Repo cleanup, bug fixes, and style guide improvements

### PR DESCRIPTION
**Pull request by laras126**
This PR does not contain changes to publicly available Larva features. It does contain changes that make it easier to test the functionality of contributions to Larva packages, and sets us up to install Larva from a Git URL vs. an npm package, eventually.

The changes include:
1. The root package.json includes packages with file paths
2. More reliable error handling in the pattern server
3. Add a-font to algorithms, and include corresponding tokens to be iterated on
4. Update style guide in sever to include data from actual tokens
5. Implement new configuration for `assets` and `brand` 
3. Keys requested via `getAppConfiguration` fallback to a default
4. Fix bugs in larva-css build, update to dart-sass, and stylelint errors
5. Add missing color tokens to base configuration
5. Enable the Larva server to be run from the root of this repo, and more the Larva file structure from packages/larva to the root
 
Most of the diff in this PR is related to the final item. The files moved are not used publicly, and are for testing running Larva scripts and building patterns in a file structure. This makes them more visible and easier to work on in a way that matches themes. 

Further, to enable installing Larva from the repo root – i.e. lessening dependency on the mono-repo structure which has been hindering, not helping us – the source of `@penskemediacorp/larva` _can_ now be the root of this repo, and packages/larva can evolve into a packages/larva-scripts, or similar. 

- [x] Static site generation is working
- [x] Tested update by npm linking to an existing Larva theme and running scripts

## To test this PR:
- Clone the repo and checkout to the v1-develop branch
- Follow instructions in the readme to install dependencies and bring up the Larva server
- Running `npm run prod:scss` and `npm run prod:js` from the repo root and ensure they work (prod:js will output linter errors - this is fixed in #308)
- Navigate to localhost:3000/larva/modules/story
- In a text editor, make a change to the value of some property in packages/larva-patterns/modules/story/story.prototype.js, or a change to packages/larva-patterns/modules/story/story.twig
- Refresh the page and confirm you see this change
- Navigate to localhost:3000/larva/style-guide
- Append localhost:3000/larva/style-guide?tokens=sportico and ensure you see an error about font tokens and the correct color tokens

### CODE REVIEW

**Comment by laras126** [comment]
@ethitter Heads up I decided to move this PR back to a draft and combine with the other style guide improvements so that they can be tested together.

**Comment by ethitter** [comment]
@laras126 Sounds good! I went through the test steps this morning and the only thing I noticed were a number of errors from Prettier when I ran `npm run prod:js`. I also reviewed most of the PR and didn't find anything of note. Let me know when it's ready for another look.

**Comment by laras126** [comment]
@ethitter Okay, you can pull this down again! I'm hands off now – the majority of this PR is the addition of copying the tokens files to multiple places so they can be referenced directly in the pattern library vs. as part of the compiled CSS.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,27 @@ Consult each package's README for a description and directions for use.
 
 Clone this repository then, from the root, run the following commands:
 
-* `npm install -g lerna`
-* `lerna bootstrap`
+```
+$ npm install
+$ ./node_modules/.bin/lerna bootstrap
```
Comment (hauvong) [inline README.md:15]: Maybe add a shell script for this? 
```suggestion
./run-first-time.sh
```

Btw, when providing example command, do not include the '$' as it is an invalid command when copy & pasted as example.

Comment (laras126) [reply README.md]: Updated, thanks!

**Review by ethitter — CHANGES_REQUESTED** [review]
One small change and this'll be ready to go. Looks great!

**Review by ethitter — APPROVED** [review]
:shipit:

**Pull request merged**